gui/wxpython/startup/utils.py

```diff
@@ -18,41 +18,48 @@
 
 import os
 import shutil
+import tempfile
+import sys
 
 
 def get_possible_database_path():
-    """Finds a path to what is possibly a GRASS Database. 
+    """Finds the directory to what is possibly a GRASS Database.
 
     Looks for directory named grassdata in the usual locations.
 
-    Returns the path as a string or None if nothing was found, so the
-    return value can be used to test if the directory was found.
+    Returns the path as a string or None if nothing was found, so the return 
+    value can be used to test if the directory was found.  
```

Comment:
Moving the word "return" is an unnecessary change and this adds white space at the end of line (which is not a good practice and now fortunately usually highlighted by Git).